### ISSUE TITLE
Docker args "x" should be specified only once

### ISSUE THREAD

**Issue by henryborchers** [jenkinsci/jenkinsfile-runner#459]
Starting with docker.io/jenkins/jenkinsfile-runner:1.0-beta-23 I'm unable to use certain arguments.  Including:

- --runWorkspace
- --file

When ever I try to use them as docker run args, I'll get the error message that should be specified only once, even though I am only using them once.

**Comment by oleg-nenashev** [comment]
Yes, these arguments can no longer be overridden in Docker images :(
They are declared in https://github.com/jenkinsci/jenkinsfile-runner/blob/master/packaging/docker/unix/jenkinsfile-runner-launcher#L19-L24, and hence attempts to declare them externally fail. A script path is needed
